05-telnyx-media-streaming/python/server.py

```
@@ -0,0 +1,152 @@
+                    await websocket.send_json(
+                        {"event": "media", "media": {"payload": message.audio}}
+                    )
+            case "conversation_created":
+                conversation_created.set()
+            case "error":
+                print(f"Phonic error: {message.error}")
+
+    async def send_to_phonic():
+        async with client.conversations.connect() as socket:
+            socket.on("message", receive_from_phonic)
+            asyncio.create_task(socket.start_listening())
```

Ruff (0.15.21)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)